src/gauravk/Assignment_6/CountDifferentContentTypes.java

```diff
@@ -0,0 +1,71 @@
+package gauravk.Assignment_6;
+
+/*
+ #6
+
+1) Count Digits, Letters, UpperCase characters, LowerCase characters, Special characters from predefined string.
+
+Input: 1rRpd3F9#K(E
+Output : 
+Total digit: 3
+Total Letters: 7
+Total Uppercase: 4
+Total Lowercase: 3
+Total Special char: 2
+ */
+
+import java.util.Scanner;
+
+public class CountDifferentContentTypes {
+	String inp = "";
+	int inputLength = 0;
+	char [] ch = new char[inp.length()];
+	int digitCount = 0;
+	int letterCount = 0;
+	int upperCaseCount = 0;
+	int lowerCaseCount = 0;
+	int specialCount = 0;
+	
+	void inputKeystrokes() {
+		Scanner sc = new Scanner(System.in);
+		String inp = sc.nextLine();
+		inputLength = inp.length();
+		ch = new char [inputLength];
+		
+		for(int index=0; index<inputLength; index++) {
+			ch[index] = inp.charAt(index);
+		}
+		countContent(ch, inputLength);
+	}
+	
+	void countContent(char [] arg, int length) {
```

Comment:
Here, we don't require to pass length of array as we can access property of array which is length.
So loop looks like bleow

for(int i =0;i<**arg.length**;i++){
   ///code
}